network/server_discovery.go

```diff
@@ -226,7 +226,7 @@ func (s *Server) setupDiscovery() error {
 	)
 
 	// Register a network event handler
-	if subscribeErr := s.SubscribeFn(context.Background(), discoveryService.HandleNetworkEvent); subscribeErr != nil {
+	if subscribeErr := s.Subscribe(context.Background(), discoveryService.HandleNetworkEvent); subscribeErr != nil {
```

Comment:
nitpick: `subscribeErr` -> `err` for consistency?

Reply:
fix https://github.com/0xPolygon/polygon-edge/pull/1593/commits/27ce806db0e450bc3d9116dfe161d06f4a897b10